rpmlint/checks/SUIDPermissionsCheck.py

```diff
@@ -0,0 +1,224 @@
+#############################################################################
+# File          : CheckSUIDPermissions.py
+# Package       : rpmlint
+# Author        : Ludwig Nussel
+# Purpose       : Check for /usr/share/permissions violations
+#############################################################################
+
+import os
+import stat
+import sys
+
+import rpm
+
+from .AbstractCheck import AbstractCheck
+from ..pkg import FakePkg
+
+_permissions_d_allowed = (
+    'lprng',
+    'lprng.paranoid',
+    'mail-server',
+    'mail-server.paranoid',
+    'postfix',
+    'postfix.paranoid',
+    'sendmail',
+    'sendmail.paranoid',
+    'squid',
+    'texlive',
+    'texlive.texlive',
+    'otrs',  # bsc#1118049
+)
```

Comment:
I bet this should be in a config file as a toml list, also any insight where this default came from?